client/src/modules/fiscal/fiscal.js

```diff
@@ -45,19 +54,18 @@ function FiscalController($state, Fiscal, ModalService, Notify, $window) {
   // switch to delete warning mode
   function del(fiscal) {
     ModalService.confirm('FORM.DIALOGS.CONFIRM_DELETE')
-    .then(function (bool) {
-       // if the user clicked cancel, reset the view and return
-      if (!bool) { return; }
-
-      Fiscal.delete(fiscal.id)
-        .then(function () {
-          Notify.success('FORM.INFO.DELETE_SUCCESS');
-          return refreshFiscalYear();
-        })
-        .catch(function (error) {
-          Notify.danger('FISCAL.CAN_NOT_DELETE_FY');
-        });
-    });
+      .then((bool) => {
```

Comment:
All of these arrow functions will need to be migrated to `function () {` calls to avoid breaking our production builds.